doc/source/workflows/basics.rst

```diff
@@ -53,26 +65,22 @@ A single line is all you need to run the workflow DAG:
     # Execute the workflow and print the result.
     print(workflow.run(output))
 
-    # You can also run the workflow asynchronously and fetching the output via 'ray.get'
+    # You can also run the workflow asynchronously and fetch the output via
+    # 'ray.get'
     output_ref = workflow.run_async(dag)
     print(ray.get(output_ref))
 
-Here this figure visualizes the workflow we created:
 
-.. image:: basic.png
-   :width: 500px
-   :align: center
-
-Each node in the original DAG becomes a workflow task.
-Workflow tasks behave similarly to Ray tasks. They are executed in a parallel and distributed way.
+Each node in the original DAG becomes a workflow task. Workflow tasks behave
+similarly to Ray tasks. They are executed in a parallel and distributed way.
 
```

Comment:
I think at this point you need to highlight the benefits of using workflows over normal Ray tasks. I don't know what they are (you mention durability and the ability to visualize the DAG). This is where you want to highlight all the benefits before going into details such as workflow options.

As a reader I'm wondering what the value of the alternate syntax is - it seems to be doing the same as before, so why do we have this alternate syntax?